automatic/sysinternals/tools/chocolateyInstall.ps1

```diff
@@ -3,32 +3,46 @@ $packageName = 'sysinternals'
 $url = 'https://live.sysinternals.com/files/SysinternalsSuite.zip'
 $installDir = Split-Path -parent $MyInvocation.MyCommand.Definition
 
-Install-ChocolateyZipPackage $packageName $url $installDir
+# Default the values
+$defaultInstallDir = Split-Path -parent $MyInvocation.MyCommand.Definition
+$installDir = $defaultInstallDir
+
+# Now parse the packageParameters using good old regular expression
+if ($packageParameters -and $packageParameters.Length -gt 18) {

```

Comment:
Why installationpath.  I was copying the Package Parameters example in the Chocolatey Wiki.  In this commit, I'm using the shortcut method to parse params, but in the future more params could be added.  In this way I (or someone else) can just use the Wiki example code straight up.